src/app/modules/reports/components/time-entries-table/time-entries-table.component.scss

```diff
@@ -26,3 +26,8 @@
 .hidden-col{
   display: none;
 }
+.table.dataTable th,
```

Comment:
@jr-98 is this style working? dataTable is not a class

Reply:
I took as an example a table that had a similar problem and the solution was that and it worked? What should I change?

Reply:
@scastillo-jp  The class does work